README.md

```diff
@@ -12,60 +12,65 @@ Page progress may also be displayed on menu items representing the page.
 
 ## Installation
 
-As one of Adapt's *[core extensions](https://github.com/adaptlearning/adapt_framework/wiki/Core-Plug-ins-in-the-Adapt-Learning-Framework#extensions),* **Page Level Progress** is included with the [installation of the Adapt framework](https://github.com/adaptlearning/adapt_framework/wiki/Manual-installation-of-the-Adapt-framework#installation) and the [installation of the Adapt authoring tool](https://github.com/adaptlearning/adapt_authoring/wiki/Installing-Adapt-Origin).
+As one of Adapt's _[core extensions](https://github.com/adaptlearning/adapt_framework/wiki/Core-Plug-ins-in-the-Adapt-Learning-Framework#extensions),_ **Page Level Progress** is included with the [installation of the Adapt framework](https://github.com/adaptlearning/adapt_framework/wiki/Manual-installation-of-the-Adapt-framework#installation) and the [installation of the Adapt authoring tool](https://github.com/adaptlearning/adapt_authoring/wiki/Installing-Adapt-Origin).
 
-* If **Page Level Progress** has been uninstalled from the Adapt framework, it may be reinstalled.
-With the [Adapt CLI](https://github.com/adaptlearning/adapt-cli) installed, run the following from the command line:
-`adapt install adapt-contrib-pageLevelProgress`
+- If **Page Level Progress** has been uninstalled from the Adapt framework, it may be reinstalled.
+  With the [Adapt CLI](https://github.com/adaptlearning/adapt-cli) installed, run the following from the command line:
+  `adapt install adapt-contrib-pageLevelProgress`
 
-    Alternatively, this extension can also be installed by adding the following line of code to the *adapt.json* file:
-    `"adapt-contrib-pageLevelProgress": "*"`
-    Then running the command:
-    `adapt install`
-    (This second method will reinstall all plug-ins listed in *adapt.json*.)
+      Alternatively, this extension can also be installed by adding the following line of code to the *adapt.json* file:
```

Comment:
Does the adapt.json need underscores?

Reply:
Just formatting, it all looks as it should when previewing the file.